Sources/Commands/SwiftPackageTool.swift

```diff
@@ -120,11 +120,13 @@ public class SwiftPackageTool: SwiftTool<PackageToolOptions> {
 
         case .initPackage:
             // FIXME: Error handling.
```

Comment:
@aciidb0mb3r Can I remove the `FIXME:` here? now I am emitting a WorkingDirNotFoundDiagnostic(). Is it enough or should I handle other things/in a different way? or would it be better to do a "PackagePathNotFoundDiagnostic()" ?

thanks in advance.